enhancements/installer/packet-ipi.md

```diff
@@ -0,0 +1,313 @@
+reservations, virtual networks, and fast direct-attached storage.
+
+Currently, users who want to [deploy OpenShift on
+Packet](https://www.packet.com/solutions/openshift/) must follow the [OpenShift
+via Terraform on
+Packet][https://github.com/RedHatSI/openshift-packet-deploy/blob/master/terraform/README.md]
+UPI instructions. This process is not streamlined and can not be integrated into
+a simple experience like the [Try OpenShift 4](https://www.openshift.com/try)
+workflow.
+
+### Goals
+
+The main goal of the Packet IPI is to provide users with an easier path to
+running Red Hat OpenShift 4 on a dynamically provisioned, horizontally scalable,
+bare metal architecture in [Packet data
+centers](https://www.packet.com/developers/docs/getting-started/datacenters/).
+
+As a first step, we would add the Packet IPI to the installer codebase. With
+accompanying documentation, this makes a the Packet option available through the
+CLI. This would enable users to deploy OpenShift with IPI in a way that is very
+similar to UPI but simplifies the process.
+
+Following other IPI installers, this first step would include
+
+- Making Packet IPI documentation available here:
+  <https://github.com/openshift/installer/blob/master/docs/user/packet/install_ipi.md>
+- Adapt Packet's sig-lifecyclce ClusterAPI v1alpha2 for use as an OpenShift's
+  ClusterAPI v1beta1 Machine driver
+- Prepare the Terraform code and Packet types necessary for an IPI installer
+- Making a CI job executing the provisioning scripts to test the IPI installer
+
+### Non-Goals
+
+It is outside the scope of this enhancement to provide details about the
+installation and infrastructure elements that have been highlighted in [Open
+Questions](#open-questions) and [Implementation
+Details/Notes/Constraints](#implementation-details-notes-constraints), even for
+those elements that are required (e.g. DNS).
+
+When open questions are explored and resolved, they will be added to this
+enhancement request or they will be surfaced in additional enhancement requests,
+as needed.
+
+## Proposal
+
+- Define and implement Packet types in the openshift/installer
+  - High level variables should include:
+    - [API Key](https://www.packet.com/developers/docs/API/) (required)
+    - [Project
+      ID](https://www.packet.com/developers/docs/getting-started/support/organizations-and-users/)
+      (required) (Options to create a project could be presented)
+  - Boostrap node variables include:
+    - Device plan (required, defaulting to minimum required)
+    - Facility (required)
+    - Virtual Network (optional, defaulted to a new virtual network)
+  - Control plane variables rely on boostrap variables and add:
+    - Device plan (defaulting to provide a good experience)
+  - Additional clusters created through the Machine/MachineSet will require
+    fields commensurate with the sig-cluster-lifecycle Packet Cluster API
+    Provider specifications
```

Comment:
`clusters` probably isn't the right word here, OCP doesn't create clusters, do you mean additional virtual machines or something?

Reply:
I meant `additional bare metal machines`. My point is that the fields that we use to define the machine spec will map closely (but independently) from the fields presented in the existing v1alpha3 Cluster API provider.

I'll reword as such.